bin/single-new-eks-mixed-observability.ts

```diff
@@ -1,4 +1,4 @@
-import SingleNewEksMixedobservabilityConstruct from '../lib/single-new-eks-mixed-observability-construct';
+import SingleNewEksMixedobservabilityConstruct from '../lib/single-new-eks-mixed-observability';
```

Comment:
Why is class name `SingleNewEksMixedobservabilityConstruct ` still having Construct ? We want to totally get rid of word Construct with Pattern. Also we dont want -construct to be removed. We would rather want it to be renamed to -pattern.